packages/e2e/cypress/support/commands.js

```diff
@@ -19,4 +19,20 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+import { escapeRegExp } from 'lodash';
+Cypress.Commands.add('getInputField', (field) => {
+  const passwordFieldNames = ['Password', 'Confirm Password'];
+  const isPasswordField = passwordFieldNames.includes(field);
+  const regexString = `^${escapeRegExp(field)}$`;
+  const regex = new RegExp(regexString, 'i');
+
+  if (isPasswordField) {
+    // TODO: we should use cy.findByLabelText once our dom is cleaned up
+    return cy.findByPlaceholderText(regex);
+  } else {
+    return cy.findByRole('textbox', { name: regex });
+  }
+});
```

Comment:
Beautiful! With my custom command about to land we can start doing:
```javascript
cy.getInputField(loginMechanism).typeAliasWithStatus(loginMechanism, status);
```
Abstraction!

Reply:
Woo! And this will hide implementation detail like `cy.findByPlaceholderText` vs `cy.findByRole`.

Reply:
I'll change the name of this function to `findInputField` to be consistent with other dom-testing-library functions.